lib/manager/poetry/__fixtures__/pyproject.10.toml

```diff
@@ -1,3 +1,18 @@
-[tool.poetry.dependencies]
-python = "^3.9"
-boto3 = "*"
+[[tool.poetry.source]]
+This conversation was marked as resolved by viceice
```

Comment:
@viceice are you seeing what I'm seeing?

![image](https://user-images.githubusercontent.com/6311784/107925742-22c31880-6f75-11eb-90e0-b16223b2aeba.png)

Reply:
already fixed 🙃